public/style/fontFamily.css

```diff
@@ -0,0 +1,2 @@
+/* INCONSOLATA */
+@import url('https://fonts.googleapis.com/css2?family=Inconsolata:wght@200;300;400&display=swap');
```

Comment:
You probably should have your fonts localy instead of getting them from google everytime (and warn google each time a user visit your site)